config.yaml

```diff
@@ -17,9 +17,9 @@ password:
 database:
 # Endpoint for scraping prometheus metrics. Default localhost:9090.
 metrics_endpoint: localhost:9090
-# Table with spans. Default "jaeger_spans_local".
+# Table with spans. Default "jaeger_spans".
```

Comment:
Why are we renaming the names to `jaeger_spans`? The embedded scripts create "local" tables and it seems to be convention to use `_local` suffix for these tables.

Reply:
Is the suffix needed if we don't use any "global" tables? I thought to add the suffix to local tables automatically if and only if we use distributed table.

Reply:
I think it's better to be explicit and use always `_local` for local tables. It's clear what it means and how it behaves.

Reply:
Okay, so be it

Reply:
then please revert this file